pom.xml

```diff
@@ -92,6 +92,24 @@
                 <artifactId>guava</artifactId>
                 <version>25.0-jre</version>
             </dependency>
+            <dependency>
+                <groupId>org.graalvm.js</groupId>
+                <artifactId>js</artifactId>
+                <version>22.0.0.2</version>
+                <scope>runtime</scope>
```

Comment:
Scopes should go in the actual dependencies section, not in the dependency manager section.